active/radiobuttons/RadioButtonsAPI.md

```diff
@@ -0,0 +1,182 @@
+# Background
+
+The RadioButtons control is our answer to enabling better keyboarding and narrator support for
+RadioButton elements that need to be presented in a grouped layout.
+
+Currently, the only way to group single RadioButton elements today is through using a StackPanel
+and individually labeling each RadioButton through its GroupName property in order for narrator to
+read out the options somewhat understandably.
+
+However, using that method still leaves gaps in the narrator experience and StackPanel does not
+behave the way users expect when navigating the list via keyboard - RadioButtons was introduced to
+help solve those issues.
+
+# Description
+
+RadioButtons is a new control that enables you to create groups of RadioButton elements easily,
+while also correctly supporting keyboarding and narrator functionality.
+
+# Examples
+
+The following example demonstrates how to create a small collective group of RadioButton elements
+using the RadioButtons control.
+
+```xaml
+<muxc:RadioButtons Header="App Mode" SelectedIndex="2">
+    <x:String>Dark</x:String>
+    <x:String>Light</x:String>
+    <x:String>Windows Default</x:String>         
+</muxc:RadioButtons>
+```
+
+![alt text](DefaultRadioButtonGroup.png)
+
+## DataBinding with RadioButtons
+An example of simple data binding of the ItemsSource with RadioButtons.
+
+```xaml
+<preview:RadioButtons Header="App Mode" ItemsSource="{x:Bind radioButtonItems}" />
+```
+
+```C#
+public sealed partial class MainPage : Page
+{
+    public class OptionDataModel
+    {
+        public string Label;
+        public override string ToString()
+        {
+            return Label;
+        }
+    }
+
+    List<OptionDataModel> radioButtonItems;
+
+    public MainPage()
+    {
+        this.InitializeComponent();
+
+        radioButtonItems = new List<OptionDataModel>();
+        radioButtonItems.Add(new OptionDataModel() { label = "Item 1" });
```

Comment:
I'm guessing you're trying to show the same app mode example as above, just using data binding. So perhaps use the dark/light strings from the previous example?